hack/build-image

```diff
@@ -125,6 +125,7 @@ QUAL_FQIN = 'fqin'
 
 
 _DISCOVERED_CONTAINER_ENGINES = []
+_BUILDX_BUILDER_NAME = "samba-in-kubernetes"
```

Comment:
Could you explain what this does? I'm not too familiar with buildx.

Reply:
When installing buildx, a default builder is created that can only build images for the host platform, which in my case is amd64.

The `docker buildx ls` command can be used to list all available builders:

```sh
NAME/NODE               DRIVER/ENDPOINT             STATUS  BUILDKIT             PLATFORMS
default *               docker                                                   
  default               default                     running v0.11.7+d3e6c1360f6e linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/386
```

To build images for platforms other than the host, a new builder must be created that supports other architectures. Each builder requires a unique name, so I used the GitHub group name as the builder name, and this variable holds its value.

After creating the new builder, samba-in-kubernetes, the output of `docker buildx ls` looks like this:

```sh
NAME/NODE               DRIVER/ENDPOINT             STATUS  BUILDKIT             PLATFORMS
samba-in-kubernetes     docker-container                                         
  samba-in-kubernetes0  unix:///var/run/docker.sock running v0.16.0              linux/arm64*, linux/amd64*, linux/arm/v7*
```

Reply:
I think I understand the buildx workflow a bit better now, thanks!

Reply:
I removed the global variable and used `target.flat_name()` as the builder name instead.